### PR TITLE
Adding season to ff league function

### PR DESCRIPTION
**Pull request by joeflan**
First time contributing to this project, let me know if I've messed anything up.

### CODE REVIEW

File: R/espn_league.R [hunk 1]
```diff
@@ -27,6 +27,7 @@ ff_league.espn_conn <- function(conn) {
   tibble::tibble(
     league_id = conn$league_id,
     league_name = league_endpoint$content$settings$name,
+    season = conn$season,
```
Comment (tanho63) [inline R/espn_league.R:30]: Looks great so far! Can you make these integers across the board for consistency?

**Review by tanho63 — APPROVED** [review]

**Review by tanho63 — CHANGES_REQUESTED** [review]
see note about ints

File: R/espn_league.R (hunk 1 above)
Comment (joeflan) [reply R/espn_league.R]: all set

**Review by tanho63 — APPROVED** [review]

**Comment by tanho63** [comment]
Looks great! Running automated checks now, then will merge/bumpver/add entry to news. It'll sit in dev for a bit until a few more minor tweaks get added, then eventually I'll make one larger release that goes to CRAN!

**Comment by joeflan** [comment]
> Looks great! Running automated checks now, then I will merge/bumpver/add entry to news. It'll sit in dev for a bit until a few more minor tweaks get added, then eventually I'll make one larger release that goes to CRAN!

Awesome. I know it's an easy one but I'm hoping I can get my feet wet with a couple of these first and then contribute a bit more.

**Pull request merged**